VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -150,6 +155,16 @@ export class ParticipantsPanelComponent implements OnInit, OnDestroy {
         );
     }
 
+    setupParticipantsSubscribers() {
+        this.participantsSubscription$.add(
+            this.participantsService.onParticipantsUpdated$.subscribe(() => {
+                const mapper = new ParticipantPanelModelMapper();
+                this.nonEndpointParticipants = this.participantsService.nonEndpointParticipants.map(x => mapper.mapFromParticipantModel(x));
```

Comment:
Can you currently not add an additional endpoint as a last minute booking?

Reply:
You cannot. And endpoints are handled as a different event.